pageserve/requirements.txt

```diff
@@ -1,2 +1,4 @@
 flask
-requests
\ No newline at end of file
+requests
+Flask-Testing
+blinker
```

Comment:
What's blinker for? Is it something Flask-Testing depends on? If so and if you need to explicitly add it to your requirements, it might be good to add a comment.

Reply:
Flask-Testing depends on it, added comment in 1bf1c5d